jfx_bridge/bridge.py

```diff
@@ -863,6 +864,23 @@ def local_eval(self, args_dict):
 
         return self.serialize_to_dict(result)
 
+    def remote_get_batch(self, handle, attrs):
+        self.logger.debug("remote_get_batch {}".format(attrs))
+
+        command_dict = {CMD: GET_BATCH, ARGS:
+            {HANDLE: handle, ATTRS: attrs}}
+        result = self.send_cmd(command_dict)
+        return self.deserialize_from_dict(result)
+
+    def local_get_batch(self, args_dict):
+        handle = args_dict[HANDLE]
+        self.logger.debug("local_get_batch {}".format(handle))
+
+        target_obj = self.get_object_by_handle(handle)
+        result = {name: getattr(target_obj, name) for name in args_dict[ATTRS]}
```

Comment:
Will throw AttributeError if that attribute isn't present.

Wrap with exception handling that will return the exception object (see how local_get does it)